### PR TITLE
fixes #5 - Added method to access loklak search API

### LINKED ISSUE #5: Add method to access loklak search api service
Presently the library does not contains method to access loklak search API service.

### PR DESCRIPTION
**Pull request by djmgit**
fixed issue #5, now the library contains method to access
loklak search API service.

### CODE REVIEW

File: lib/loklak.rb [hunk 1]
```diff
@@ -24,4 +24,31 @@ def self.status()
       {}
     end
   end
+
+  def self.search(query=nil, since_date=nil, until_date=nil, from_user=nil, count=nil)
+    method_path = 'search.json?'
```
Comment (sudheesh001) [inline lib/loklak.rb]: Remove the `?` from here and put it before `query=` That makes more sense than having a query endpoint with a `?` trailing it.

File: lib/loklak.rb [hunk 2]
```diff
@@ -24,4 +24,31 @@ def self.status()
...
+    if query != nil
+      method_path += 'query=' + query
+      if since_date != nil
+        method_path += '%20since:' + since_date
+      end
+      if until_date != nil
+        method_path += '%20until:' + until_date
+      end
+      if from_user != nil
+        method_path += '%20from:' + from_user
```
Comment (sudheesh001) [inline lib/loklak.rb]: Please use ` ` in place of `%20` You can encode the URLs using ruby's libraries

**Review by sudheesh001 — CHANGES_REQUESTED** [review]

**Comment by djmgit** [comment]
@sudheesh001 made the changes

**Review by daminisatya — APPROVED** [review]

**Comment by djmgit** [comment]
@daminisatya @sudheesh001 should I make any other changes in this PR?

**Review by sudheesh001 — APPROVED** [review]

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by djmgit** [comment]
@mariobehling resolved conflicts

**Pull request merged**